profiling/run_doc_profiling_plots.sh

```
@@ -226,14 +229,25 @@ python profiling/pipeline/analyse_memory.py \
+	--labels 100 500 1000 5000 10000 100000 \
+	--output-dir "$MEMORY_ANALYSIS_DIR"
+
 echo ""
 echo "========================================"
 echo "Particle and Wavelength Scaling"
 echo "========================================"
 
 # Run the general profiling scripts using make_all_plots.py
 echo "Running particle and wavelength scaling profiling..."
-python profiling/general/make_all_plots.py --nthreads $SCALING_THREADS --n_averages 3
+python profiling/general/make_all_plots.py --nthreads $SCALING_THREADS --n_averages 3 --output_dir "$FINAL_PLOTS_DIR"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)